mne/tests/test_annotations.py

```diff
@@ -247,8 +287,8 @@ def test_crop_more():
     assert len(raw_concat.annotations) == 4
     assert_array_equal(raw_concat.annotations.description,
                        raw.annotations.description)
-    assert_allclose(raw.annotations.duration, duration)
-    assert_allclose(raw_concat.annotations.duration, duration)
+    assert_allclose(raw.annotations.duration, duration, atol=1e-6)
+    assert_allclose(raw_concat.annotations.duration, duration, atol=1e-6)
```

Comment:
is this necessary? do we have a loss of precision with this change?

Reply:
This one I can push a little fix for